### DISCUSSION TITLE
How can I change the color of these elements? (marked with arrows)

### DISCUSSION THREAD

**Discussion by pojerateldush** [aXenDeveloper/ips-theme-dynamic#9] [Q&A]
![image](https://user-images.githubusercontent.com/81668134/159573551-2fe10944-4640-465a-98a2-6a4a6e08bb78.png)

I can't figure out how to change it. I want to put green everywhere.
There are no other problems, the theme is very good!

**Comment by aXenDeveloper** [comment]
```
--theme-dynamic-gradient-primary: linear-gradient(90deg, rgb( var(--theme-dynamic-color-1) ), rgb( var(--theme-dynamic-color-2 )));
```
